src/main/resources/defaults/data/Banners.json

```diff
@@ -11,7 +11,8 @@
 		"endTime": 1924992000,
 		"sortId": 1000,
 		"fallbackItems4Pool1": [1006, 1014, 1015, 1020, 1021, 1023, 1024, 1025, 1027, 1031, 1032, 1034, 1036, 1039, 1043, 1044, 1045, 1048, 1053, 1055, 1056, 1064],
-		"weights5": [[1,75], [73,150], [90,10000]]
+		"weights5": [[1,75], [73,150], [90,10000]],
+        "lootTable": "gacha/StandardPool.json"
```

Comment:
This seems like a weird one if I understand it correctly, as the `lootTable` would override `weights5` and `fallbackItems4Pool1`.

Reply:
Yes. if `lootTable` is set, all fields describing the output of the banner will be ignored since the loot table take the control of it

Reply:
cleaned up unused fields